RELEASES.md

```diff
@@ -100,9 +100,10 @@ The current state is available in the following tables:
 | [1.1](https://github.com/containerd/containerd/releases/tag/v1.1.8)  | End of Life | April 23, 2018  | October 23, 2019 |
 | [1.2](https://github.com/containerd/containerd/releases/tag/v1.2.13) | End of Life | October 24, 2018 | October 15, 2020 |
 | [1.3](https://github.com/containerd/containerd/releases/tag/v1.3.10) | End of Life | September 26, 2019  | March 4, 2021 |
-| [1.4](https://github.com/containerd/containerd/releases/tag/v1.4.12) | Extended   | August 17, 2020 | November 3, 2021 (Active), March 3, 2022 (Extended) |
-| [1.5](https://github.com/containerd/containerd/releases/tag/v1.5.9)  | Active   | May 3, 2021  | max(May 3, 2022, release of 1.6.0 + 6 months) |
-| [1.6](https://github.com/containerd/containerd/milestone/34)         | Next   | TBD  | max(TBD+1 year, release of 1.7.0 or 2.0.0 + 6 months) |
```

Comment:
How many 1.x releases will we have before v2.0

Reply:
I removed the mentioned of later versions since that isn't decided. I think 1.7 should be the last 1.x release so did not want to add a mention of a possible 1.8. I was thinking of discussing this some at the Thursday community meeting.

Reply:
what we going to do in 2.0?

Reply:
🪓

Reply:
> what we going to do in 2.0?

I just want to remove deprecated stuffs (runtime v1, runtime v2 runc v1, config v1, ...)